src/alarm_trap_sender.cpp

```diff
@@ -143,3 +166,112 @@ void AlarmTrapSender::send_trap(const AlarmTableDef& alarm_table_def)
 
   snmp_reset_var_buffers(&var_trap);
 }
+
+void AlarmTrapSender::send_enterprise_trap(const AlarmTableDef& alarm_table_def)
+{
+  static const unsigned int severity_to_state[] = {2, 1, 2, 6, 5, 4, 3};
+  static const oid snmp_trap_oid[] = {1,3,6,1,6,3,1,1,4,1,0};
+  static const oid trap_type_oid[] = {1,2,826,0,1,1578918,19444,9,2,1,1};
```

Comment:
I don't think this is right - "1,2,826,0,1,1578918" is dataConnection in the MIB, whereas dataConnectionTraps is "enterprises.19444", i.e. 1.3.6.1.4.1.9.19444. This mismatch means that Wireshark can't parse these traps.